frontend/src/components/config/App.tsx

```diff
@@ -17,6 +18,7 @@ function App() {
       <CustomRoute path="/game" component={GamePage} layout={GameLayout} exact />
       <CustomRoute path="/game/join" component={JoinGamePage} layout={MainLayout} exact />
       <CustomRoute path="/game/create" component={CreateGamePage} layout={MainLayout} exact />
+      <CustomRoute path="/game/waiting" component={WaitingRoomPage} layout={MainLayout} exact />
```

Comment:
Personal preference and nit, but what do you think about calling this '/game/lobby' or something like that instead?

Reply:
I was thinking that `waiting` wasn't the best name; `lobby` is much better, thanks I'll change it to that!

Reply:
Done!